### DISCUSSION TITLE
能问一个问题吗？你们游戏都是哪里下的？

### DISCUSSION THREAD

**Discussion by owl342655596** [Gamuxorg/bbs#304] [建议和闲聊]
你们打包的那些游戏都是哪里找的？

**Comment by shouhuanxiaoji** [comment]
很多，游戏网站，rutracker gog steam github等

**Comment by hyh19962008** [comment]
朋友有兴趣一起打包游戏造福更多玩家不？半小时就能上手的

Best

________________________________
